modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc

```diff
@@ -3,9 +3,454 @@
 :navtitle: ACID Transactions
 :page-topic-type: howto
 
+include::project-docs:partial$attributes.adoc[]
+
 [abstract]
 {description}
 
+include::{version-server}@sdk:shared:partial$acid-transactions.adoc[tag=intro]
+
+
+== Requirements
+
+* Couchbase Server 6.6.1 or above.
+* Couchbase PHP SDK 4.0.0 or above.
+include::{version-server}@sdk:shared:partial$acid-transactions.adoc[tag=requirements]
+
+
+== Getting Started
+
+Couchbase transactions require no additional components or services to be configured. 
+Simply install the most recent version of the SDK.
+// You may, on occasion, need to import some enumerations for particular settings, but in basic cases nothing is needed.
+
+== Configuration
+
+Transactions can optionally be globally configured when configuring the `Cluster`.
+For example, if you want to change the level of durability which must be attained, this can be configured as part of the connect options:
+
+[source,php]
+----
+include::example$transactions-example.php[tag=config,indent=0]
+----
+
+include::{version-server}@sdk:shared:partial$acid-transactions.adoc[tag=config]
+
+// TODO: this partial needs to change a bit, owing to library references.
+include::{version-server}@sdk:shared:partial$acid-transactions.adoc[tag=creating]
```

Comment:
This TODO can be removed, library references have been cleaned up from the `docs-sdk-common` partials.
https://github.com/couchbase/docs-sdk-common/blob/release/7.1/modules/shared/partials/acid-transactions.adoc

Reply:
I think this is probably based off an older version of the Node.js transactions page(it was updated recently to address alot of these todos). I also updated `docs-sdk-common` with Graham to ensure the partials are no longer Java specific.

Have a look at the latest Node.js page, that should help update this page a little bit.
https://github.com/couchbase/docs-sdk-nodejs/blob/release/4.0/modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc